packages/payload/src/admin/components/forms/field-types/DateTime/index.tsx

```diff
@@ -60,9 +73,9 @@ const DateTime: React.FC<Props> = (props) => {
       }}
     >
       <div className={`${baseClass}__error-wrap`}>
-        <Error message={errorMessage} showError={showError} />
+        <ErrorComp message={errorMessage} showError={showError} />
       </div>
-      <Label htmlFor={path} label={label} required={required} />
+      <LabelComp htmlFor={path} label={label} required={required} />
       <div className={`${baseClass}__input-wrapper`} id={`field-${path.replace(/\./g, '__')}`}>
         <DatePicker
```

Comment:
You don't think we could add render the `BeforeInput` or `AfterInput` components around the `<DatePicker>`?

Reply:
Initially, I thought that the `<DatePicker/>` itself is a complex component and `BeforeInput` and `AfterInput` should be used only in fields that have `input` or `textarea`. But thinking now, it may be used to place tooltips.